spec/integration/guest_view_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'spec_helper'
+
+describe 'the guest view', type: :feature do
+  #include Capybara::DSL
+
+  describe 'the home page' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.